net/wrk/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           compilers 1.0
 PortGroup           xcode_workaround 1.0
 PortGroup           openssl 1.0
 
-github.setup        wg wrk 4.1.0
+github.setup        wg wrk 4.2.0
 revision            2
```

Comment:
`revision` needs to reset to zero when the version is bumped

Reply:
@herbygillot thanks for the feedback! just pushed a commit to fix that.